src/lib/formatters/iac-output/v2/issues-list.ts

```diff
@@ -12,9 +12,18 @@ export function getIacDisplayedIssues(
   results: FormattedResult[],
   outputMeta: IacOutputMeta,
 ): string {
+  let output = EOL + colors.info.bold('Issues') + EOL;
+
   const formattedResults = formatScanResultsNewOutput(results, outputMeta);
 
-  let output = EOL + colors.info.bold('Issues') + EOL;
+  if (![...Object.keys(formattedResults.results)].length) {
```

Comment:
I might be missing something obvious, but why do we need the spread operator here?
couldn't we just check this in the condition? 
 
```suggestion
  if (!(formattedResults.results).length) {
```

Reply:
Also, I've seen that we have `lodash.isEmpty` already in the CLI, we could just check the object with that if it's more readable.